handbook/engineering/hiring/software-engineer-frontend.md

```diff
@@ -1,19 +1,34 @@
 # Software Engineer - Frontend
 
-We are looking for frontend engineers who know how to build intuitive user experiences and APIs to make the power of [Universal Code Search](https://about.sourcegraph.com/product) accessible to everyone. You will have a lot of ownership to solve tough technical and UX problems in key areas of our web application and browser extensions (e.g. code search, code navigation, campaigns).
+We are looking for frontend engineers who know how to build intuitive user experiences to make the power of [Universal Code Search](https://about.sourcegraph.com/product) accessible to everyone. You will have a lot of ownership to solve tough technical and UX problems in key areas of our web application and browser extensions (e.g. code search, code navigation, campaigns).
+
+## What you will do
+
+- Work closely with a dedicated designer and PM to implement high-quality UIs.
+- Actively participate in the planning of the features we're going to build, e.g. by writing and reviewing RFCs.
+- Build UIs you will love to use yourself as a developer.
+- Ensure UI/UX consistency in our app.
+- Identify potential for abstraction and build components that can be reused, even by other teams.
+- Improve the code patterns in our frontend.
+- Hold up accessibility standards.
 
 ## Qualifications
 
-- Excellent knowledge of TypeScript.
-- Solid understanding of JavaScript's concurrency model and event loop.
-- Skilled at building and testing (e.g., unit testing, automated end-to-end testing) React single page applications and HTTP APIs (GraphQL and Go preferred).
+- Experience in writing [SPAs](https://en.wikipedia.org/wiki/Single-page_application) using React or similar component-based frontend frameworks.
+- Familiarity with TypeScript or a different typed programming language.
+- Knowledge of CSS.
+- Knowledge of semantic HTML.
+- Experience in using backend APIs from the frontend.
+- Skilled at building and testing (e.g., unit testing, automated end-to-end testing) UIs.
 - Ability to communicate clearly and empathetically, especially in writing and documentation.
-- Practiced at creating high quality software balanced with a pragmatic understanding of how to make appropriate tradeoffs (e.g. cut scope) to ship quickly and iterate when necessary.
+- Practice at creating high quality software balanced with a pragmatic understanding of how to make appropriate tradeoffs (e.g. cut scope) to ship quickly and iterate when necessary.
+- [High agency](../../../company/values.md#High_agency).
```

Comment:
While it is true that we are looking for these things, I suspect that these don't belong under qualifications. See how I handled this for the code intel manager role (it isn't perfect, but I think it is an improvement): https://about.sourcegraph.com/handbook/engineering/hiring/engineering-manager-code-intelligence

As soon as https://github.com/sourcegraph/about/pull/1738 is merged I think both the code intel manager description and this one could be updated to point to that (but I still don't think that would be under "qualifications"). 

In my mind, qualifications should be concrete facts that a hypothetical recruiter could easily screen for when sourcing candidates or reviewing applications.

Reply:
So "Qualifications" means more like "Qualifications to get an interview"? I interpreted it as "Qualifications to get the job", which may be where the confusion came from.
Are you referring to both "High agency" and "Practice at creating high quality software" (which was there before)?

When I thought about what are the most important attributes we need in someone for this role, i.e. what is the Eigenproperty that distinguishes those who will be successful in this role from those who would be unsuccessful, high agency was the clear answer to me. I then realized that we have this as a value now with a great description with more detail, so that's why I linked to it here. It's maybe not the easiest to screen for, but I think it's definitely interviewed for in the Technical Experience interview, which is why I saw it as a qualification.

Reply:
> So "Qualifications" means more like "Qualifications to get an interview"? I interpreted it as "Qualifications to get the job", which may be where the confusion came from.

@withdavidli how do you think about this question?


Regardless of David's answer, I am happy to merge this as-is since we can update it later.

Reply:
Qualifications to get an interview. Concrete set of skills and experience that are identifiable in a candidate's profile or application. I think about this section as a subset of everything we need for the role. This section excluding anything that would be a given for any of our roles. For example: good communication skills or finishing tasks on time. 

Best use of this section is to list skills and experiences that are unique to the role that is quantifiable and identifiable. 

In these two examples, and while I understand they likely exist in current job description, they are not necessary the best for screening in or out candidates in their applications. High agency is something we would look for in all our roles, and creating high quality software is something we would look for in all our engineers.

Reply:
Cool, that's very helpful, thanks! This is great because it gave me enough items to put together an "About you" section similar to the codeintel role, with some of the other "Qualifications" moved there, reworded. I'm happy with this result, lmk if this is better now.